website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
You can't set new properties :/ I ran into that wall for awhile before figuring out to use `.toJSON`